(Android) Publish v4 to Maven

From the documentation, it looks like the latest version is v4.0.1

```
dependencies {
  implementation 'com.braintreepayments:popup-bridge:4.0.1'
}
```

However the latest published version available on Maven is v3.1.1: https://mvnrepository.com/artifact/com.braintreepayments/popup-bridge

So the above dependency resolution fails:

```
> Task :app:mergeDebugResources FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:mergeDebugResources'.
> Could not resolve all files for configuration ':app:debugRuntimeClasspath'.
   > Could not find com.braintreepayments:popup-bridge:4.0.1.
     Searched in the following locations:
       - https://dl.google.com/dl/android/maven2/com/braintreepayments/popup-bridge/4.0.1/popup-bridge-4.0.1.pom
       - https://repo.maven.apache.org/maven2/com/braintreepayments/popup-bridge/4.0.1/popup-bridge-4.0.1.pom
```         


Am I missing something?
Or are you able to publish the latest v4 to Maven to resolve this?

Thanks for your effort on this library - it's a really useful solution.

--- a/README.md
+++ b/README.md
@@ -25,7 +25,7 @@ Add the dependency in your `build.gradle`:
 
 ```groovy
 dependencies {
-  implementation 'com.braintreepayments:popup-bridge:4.0.1'
+  implementation 'com.braintreepayments.api:popup-bridge:4.0.1'
 }
 ```
 
@@ -55,7 +55,8 @@ Quick Start
        private WebView mWebView;
 
        @Override
-       public void onCreate() {
+       protected void onCreate(@Nullable Bundle savedInstanceState) {
+           super.onCreate(savedInstanceState);
            // Connect your web view.
            // ...
 

--- a/v4_MIGRATION.md
+++ b/v4_MIGRATION.md
@@ -45,7 +45,9 @@ class MyWebViewActivity extends Activity {
   private PopupBridgeClient popupBridgeClient;
   
   @Override
-  public void onCreate() {
+  protected void onCreate(@Nullable Bundle savedInstanceState) {
+    super.onCreate(savedInstanceState);
+    
     popupBridgeClient = new PopupBridgeClient(this, webView, "my-custom-url-scheme");
   
     // register error listener

